CMakeLists.txt

```diff
@@ -92,4 +92,31 @@ if(BUILD_CPR_TESTS)
     add_subdirectory(test)
 endif()
 
-install(DIRECTORY include/cpr DESTINATION include)
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+set(CPR_CONFIG_INSTALL_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
+
+install(
+  TARGETS ${PROJECT_NAME}
```

Comment:
Please do not rely on ${PROJECT_NAME} because CMake 2.8 😒
please use ${CPR_LIBRARIES} instead